p2p/simulations/adapters/inproc.go

```diff
@@ -93,23 +92,12 @@ func (s *SimAdapter) NewNode(config *NodeConfig) (Node, error) {
 		}
 	}
 
-	// dialer in simulations based on ENR records
-	// doesn't work unless we explicitly set localhost record
-	ip := enr.IP(net.IPv4(127, 0, 0, 1))
-	config.Record.Set(&ip)
-	tcpPort := enr.TCP(0)
-	config.Record.Set(&tcpPort)
-
-	err := enode.SignV4(&config.Record, config.PrivateKey)
-	if err != nil {
-		return nil, fmt.Errorf("unable to generate ENR: %v", err)
-	}
-	nod, err := enode.New(enode.V4ID{}, &config.Record)
+	// set up ENR record
```

Comment:
I don't think we need this comment or the method name should be more revealing. I prefer removing comment.